ros2pkg/ros2pkg/verb/create.py

```diff
@@ -100,21 +103,35 @@ def main(self, *, args):
             if not maintainer.email:
                 maintainer.email = maintainer.name + '@todo.todo'
 
-        cpp_node_name = None
-        if args.cpp_node_name:
-            cpp_node_name = args.cpp_node_name
-            if args.cpp_node_name == args.cpp_library_name:
-                cpp_node_name = args.cpp_node_name + '_node'
+        node_name = None
+        library_name = None
+        if args.library_name:
+            library_name = args.library_name
+        if args.node_name:
+            node_name = args.node_name
+            if args.node_name == args.library_name:
+                node_name = args.node_name + '_node'
                 print('[WARNING] node name can not be equal to the library name', file=sys.stderr)
-                print('[WARNING] renaming node to %s' % cpp_node_name, file=sys.stderr)
+                print('[WARNING] renaming node to %s' % node_name, file=sys.stderr)
 
         buildtool_depends = args.build_type
-        if args.build_type == 'ament_cmake' and args.cpp_library_name:
+        if args.build_type == 'ament_cmake' and args.library_name:
             buildtool_depends = 'ament_cmake_ros'
 
         test_dependencies = []
         if args.build_type == 'ament_cmake':
             test_dependencies = ['ament_lint_auto', 'ament_lint_common']
+        if args.build_type == 'ament_python':
+            test_dependencies = ['ament_copyright', 'ament_flake8', 'ament_pep257',
+                                 'python3-pytest']
+
+        if args.build_type == 'ament_python':
+            if args.package_name == 'test':
```

Comment:
Please merge these two lines into a single one.

Reply:
Fixed in 002ee40.